### ISSUE TITLE
[syncfusion_flutter_gauges] Does not compile in flutter version 2.11.0-0.0.pre.634

### ISSUE THREAD

**Issue by fodil-a** [syncfusion/flutter-widgets#595]
It does not compile anymore because there was a breaking change in Flutter.

Getting the widget of an Element is now a `Widget` instead of a `InheritedWidget`, because it is a performance issue.

Because of that change, it won't compile here anymore:
https://github.com/syncfusion/flutter-widgets/blob/194afe6f24944c0d998bf2edc92e1a70e851e036/packages/syncfusion_flutter_gauges/lib/src/linear_gauge/gauge/linear_gauge_scope.dart#L38-L40

It should be `Widget` instead.

Change on Flutter side is [here](https://github.com/flutter/flutter/pull/97822/commits/697afc25c4b56a8c764921e2ceceaac9d9bebeb3#diff-134bc36bd7c045a92ffff29f7efdaa7d375384911475a40343814c76c87b17cbR3235-R3242)

**Comment by PraveenGopalsamy** [comment]
Hi fodil-a,
 
We have checked the mentioned query with our [latest Flutter Gauge version](https://pub.dev/packages/syncfusion_flutter_gauges/versions) in the Flutter Stable channel. We cannot reproduce the mentioned issue from our side. Could you please confirm the version and channel which you are using. This will help us to assist you in a better way by reproducing the exact same scenario which you are facing. 
 
Please let us know if you need any further assistance in this. 
 
Regards, 
Praveen G.

**Comment by fodil-a** [comment]
Hi,

It's on master

**Comment by PraveenGopalsamy** [comment]
Hi fodil-a,
 
We would like to let you know that Syncfusion Flutter packages supports the Flutter stable channel only. Once these changes from the master branch have been moved to stable, we will resolve it immediately and publish it in the upcoming weekly release.
 
Please let us know if you need any further assistance in this. 
 
Regards,
Praveen G,

**Comment by fodil-a** [comment]
Yeah understood, it's sad that we already have the change for 2 occurrences on current code, that I did a PR to fix the third and that we won't get it 😭

Thanks you can close it, I will just use my fork for now

**Comment by PraveenGopalsamy** [comment]
Hi Fodil-a,

Thank you. If you need any assistance let us know, we will be happy to assist.

Regards,
Praveen G.
